LEGO1/mxbitmap.h

```diff
@@ -1,13 +1,30 @@
 #ifndef MXBITMAP_H
 #define MXBITMAP_H
 
-class MxBitmap
+#include <stdlib.h>
+
+#include "legoinc.h"
+#include "mxcore.h"
+#include "mxpalette.h"
+
+class MxBitmap : public MxCore
 {
 public:
   __declspec(dllexport) MxBitmap();
   __declspec(dllexport) virtual ~MxBitmap();
   __declspec(dllexport) virtual MxPalette *CreatePalette();
-  __declspec(dllexport) virtual long Read(const char *);
+  __declspec(dllexport) virtual long Read(char const *);
```

Comment:
Order matters, `Read` comes before `CreatePalette`. And perhaps also add `FIXME`'s for the missing virtual methods.